modules/vba_documents/app/models/vba_documents/upload_submission.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module VBADocuments
+  class UploadSubmission < ActiveRecord::Base
+    include SetGuid
+
+    # TODO: Persist this? Otherwise it regenerates with new expiry
```

Comment:
Worth calling out, but since the location is only rendered for the initial post, not considering it a blocker.